pkg-purge

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+{ #prevents errors if script was modified while in use
+set -o pipefail #if any command within a pipe fails, make the whole pipe fail. Necessary for error-catching with reduceapt
+DIRECTORY="$(readlink -f "$(dirname "$0")")"
+trap "exit 1" TERM
+export TOP_PID=$$
+
+error() {
+  echo -e "\e[91m$1\e[39m" 1>&2
+  kill -w -s TERM $TOP_PID
+  exit 1
+}
+
+PURGE_LIST="$1" #quotation-marked, space-seperated list of package names to install
+app="$(basename "$2")" #remove any slashes to just get program name
+
+reduceapt() { #remove unwanted lines from apt output
```

Comment:
I think it's better to declare `reduceapt` only once in `api`. that makes changing it much easier.